src/global/authentication/api.js

```diff
@@ -48,15 +49,16 @@ const api = {
 
   getUser() {
     const access_token = localStorage.getItem("access_token");
-
     return axios.get("/user", {
       params: {},
-      headers: { Authorization: `Bearer ${access_token}` },
-    });
+      headers: {
+        Authorization: `Bearer ${access_token}`
+      },
+    })
   },
 
   logout() {
-    //localStorage.clear();
+    localStorage.clear();
```

Comment:
Just use removeItem() function to remove the token because, the only safe method is individual removal. If there are other parts of your code using localstorage, you can lose your other values on localstorage.